data/file-lists/zs-tools/net_config.py

```
@@ -72,6 +72,40 @@ def execute_command(cmd_template, *args):
+    except OSError:
+        return False
+    return os.path.basename(target) == 'bnxt_en'
+
+
+def enable_promisc_for_bnxt_en_nics():
+    # http://jira.zstack.io/browse/ZSTAC-81287
+    # enable promisc for all bnxt_en nics
+    net_root = '/sys/class/net'
+    try:
+        nic_names = os.listdir(net_root)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)